tests/Unit/Property/SelectorTest.php

```diff
@@ -141,4 +141,29 @@ public function isValidForInvalidSelectorReturnsFalse(string $selector): void
     {
         self::assertFalse(Selector::isValid($selector));
     }
+
+    /**
+     * @test
+     */
+    public function cleanupSpacesWithinSelector(): void
+    {
+        $selector = <<<'CSS'
```

Comment:
Please don't use heredoc - indented heredoc doesn't work with with PHP 7.2 (though it works with 7.3 upwards), and it makes autoformatting the code more of a hassle.

Reply:
Instead, could you please switch to a literal, quotes-enclosed string?

Reply:
Thank you for bringing this to my attention, it's fixed.